flowzone.yml

```diff
@@ -2251,9 +2251,10 @@ jobs:
             exit 1
           fi
 
+          # here the number of required approvals sent to the gh API is checked back from the PUT response from GH api 
+          # Specifies the number of reviewers required to approve pull requests. Use a number between 1 and 6 or 0 to not require reviewers (successful checks => merge) .
           if [[ $(echo "$result" | jq '.required_pull_request_reviews.required_approving_review_count') -ne ${{ inputs.required_approving_review_count }} ]]; then
-            echo ::error title={url}::"branch protection required_pull_request_reviews.required_approving_review_count not set to ${{ inputs.required_approving_review_count }}"
-            exit 1
+            echo ::warning title={url}::"Failed to set required approvers count. Check that branch protection is enabled on your repo and try again."
           fi
```

Comment:
@ab77 recently removed the custom title code from a few other warnings/errors.

What purpose does it serve to set title to `{url}`? What text is this supposed to resolve to?

Reply:
@klutchell I think it was just a copy/paste approach I'll clean this up and delete the title {url}.